docs/content/preview/migrate/performance.md

```diff
@@ -70,6 +70,8 @@ These performance optimizations apply whether you are importing data using the y
 
 By default, yb-voyager exports four tables at a time. To improve data export, parallelize the export of data from multiple tables using the [–-parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument with the export data command to increase the number of jobs. Setting the value too high can however negatively impact performance; a setting of '4' typically performs well.
 
+If you use EXPERIMENTAL_FAST_EXPORT to [accelerate data export](../migrate-steps/#accelerate-data-export-for-mysql-and-oracle), `--parallel-jobs` will not have any impact on performance and yb-voyager will export only one table at a time.
+
 ## Test results
```

Comment:
@aishwarya24 A small restructure required here: 

Please move the `Test Results` section to under [Improve import performance > Techniques to improve performance](https://deploy-preview-17505--infallible-bardeen-164bc9.netlify.app/preview/migrate/performance/#techniques-to-improve-performance) (just above the note) introduction: `Performance test results to demonstrate the above techniques:`